data/json/vehicles/utility.json

```diff
@@ -231,6 +231,7 @@
             { "x":  0, "y":  0, "part": "controls" },
             { "x":  0, "y":  0, "part": "horn_big" },
             { "x":  1, "y":  0, "part": "diesel_engine_i6" },
+			{ "x":  1, "y":  0, "part": "alternator_truck"},
```

Comment:
Thanks, I can't fix in web browser so I'll submit in a little bit.  I want to go through all the engines and make sure no others are missing an alternator or battery